### ISSUE TITLE
Plans to make source available ?

### ISSUE THREAD

**Issue by boxerab** [GPUOpen-Tools/Radeon-GPUProfiler#1]
Looks like a great tool. Given the trend at AMD, I was surprised that it is only
a binary release.

**Comment by shmerl** [comment]
Any progress on open sourcing the profiler?

**Comment by pemgithub** [comment]
I agree.  It's weird that they open source other tools but not RGP?  Or if there's something AMD doesn't want to be public, maybe they could open source most of RGP, and keep some small part of it closed source (as a prebuilt binary)?

**Comment by baryluk** [comment]
I just saw Radeon Memory Visualiser just open sourced. As  Linux user, I think I would even be more happy with RGP being open sourced. It already works partially with Mesa RADV even, but open sourcing stuff, would make progress even better, not to mention maybe allow it to be included in many distros easier. I used RGP few times, and I really found it useful.

**Comment by bok11** [comment]
I am also in need of the source code for the profiler, as i need it in a different distro than ubuntu.
Is there any plans on making the source available?

**Comment by dagmcr** [comment]
any update on this?

**Comment by JLP** [comment]
Any news about making the code free and open?

**Comment by 0xfadead** [comment]
Any news about this?
